internal/git/cmdgit/git.go

```diff
@@ -126,7 +126,7 @@ func (g *Git) BranchExist(remoteName, branchName string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	return strings.Contains(stdOut, fmt.Sprintf("refs/heads/%s", branchName)), nil
+	return strings.Contains(stdOut, fmt.Sprintf("refs/heads/%s\n", branchName)), nil
```

Comment:
I think a `\t` in the beginning should be added as well. But I think we could the test for it.
![image](https://github.com/lindell/multi-gitter/assets/284868/a8bb159f-4e30-40a4-aaa1-04ffbc422ac3)

Reply:
Updated the code to also include the tab, and force pushed the branch with two separate commits (one with the test and one with the fix).